README.md

```diff
@@ -21,10 +21,7 @@ To assess the behavior of microservice versions, iter8 supports a few metrics ou
 
 ## Supported environments
 
-The _iter8-controller_ currently supports the following Kubernetes-based environments, whose traffic-management capabilities are used:
-
-* [Istio service mesh](https://istio.io)
-* [Knative](https://knative.dev)
+The _iter8-controller_ currently uses the [Istio service mesh](https://istio.io) traffic management capabilities to automate the user traffic split across multiple microservice versions.
 
 ## Installing iter8
```

Comment:
The reference to knative in this section should be removed.

Reply:
Done.